test/cyclonus/test-cyclonus-windows.sh

```diff
@@ -1,5 +1,9 @@
-curl -fsSL github.com/mattfenwick/cyclonus/releases/latest/download/cyclonus_linux_amd64.tar.gz | tar -zxv
-./cyclonus_linux_amd64/cyclonus generate \
+# curl -fsSL github.com/mattfenwick/cyclonus/releases/latest/download/cyclonus_linux_amd64.tar.gz | tar -zxv
+# ./cyclonus_linux_amd64/cyclonus generate \
+git clone https://github.com/huntergregory/cyclonus.git
+cd cyclonus/
+git checkout sleep-and-retry-after-failing
```

Comment:
You should be able to tighten the two git commands to:

```bash
git clone --branch sleep-and-retry-after-failing --depth=1 https://github.com/huntergregory/cyclonus.git
```

`--depth=1` is always useful in scripts where you don't care about the history. It just says "give me the last commit on that branch only, don't download N years of history."

Reply:
thanks for the tip!